orderer/consensus/etcdraft/eviction.go

```diff
@@ -60,6 +61,9 @@ func (pc *PeriodicCheck) check() {
 }
 
 func (pc *PeriodicCheck) conditionNotFulfilled() {
+	if pc.ReportCleared != nil && pc.conditionHoldsSince != (time.Time{}) {
```

Comment:
maybe instead of:
`pc.conditionHoldsSince != (time.Time{}`
do:
`pc.conditionHoldsSince.IsZero()` ?

Reply:
Sure -- that is quite a bit more graceful, will fix.